auth/system-test/auth.test.js

```diff
@@ -35,15 +35,15 @@ before(() => {
 });
 
 it('should load credentials implicitly', async () => {
-  const output = await tools.runAsync(`${cmd} auth-cloud-implicit`, cwd);
+  const output = await childProcess.execSync(`${cmd} auth-cloud-implicit`, cwd);
```

Comment:
`execSync` does not return a promise, so we don't want/need `await`. It's blocking by default.

Reply:
perfect, thanks!